caravel/assets/javascripts/SqlLab/components/TableWorkspaceElement.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { ButtonGroup } from 'react-bootstrap';
+import Link from './Link';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as Actions from '../actions';
+import shortid from 'shortid';
+
+// CSS
+import 'react-select/dist/react-select.css';
+
+class TableWorkspaceElement extends React.Component {
+  selectStar() {
+    let cols = '';
```

Comment:
we could simplify this block with some es6 syntax and `.map` and `.join`...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

`const cols = this.props.table.columns.map((c) => c.name).join(', ');`